report/MVP/MVP.tex

```diff
@@ -0,0 +1,23 @@
+This chapter will talk about what our Minimum Viable Product (MVP) is, and we will in the following chapters go though how we have developed this MVP.
+
+\section{What is our MVP?}
+An MVP is a piece of product, that gives some sort of value, with the least amount of work went into it.
+It will not have many advanced features, but just showcase the basic functionalities, so the product can be tested, and further be worked on, to get all of the wanted features.
+This will give some sort of idea of how good the product will be, and if the product fails, not much time is lost, because of the amount of time working on it. \cite{whatIsMVP}
```

Comment:
```suggestion
This will give the costumer an idea of how good the product will be, and may also give some early signs to the success of the product. 
If the product fails, not much time is lost, because of the amount of time working on it. \cite{whatIsMVP}
```

Reply:
```suggestion
This will give the customer an idea of how good the product will be, and may also give some early signs to the success of the product. 
If the product fails, not much time is lost, because of the amount of time working on it. \cite{whatIsMVP}
```